invokeai/app/services/events.py

```diff
@@ -26,7 +25,7 @@ def emit_generator_progress(
         self,
         graph_execution_state_id: str,
         invocation_dict: dict,
-        source_id: str,
+        source_node_id: str,
```

Comment:
Same comment as elsewhere - this should use the execution graph's node id, not the source node id.